### ISSUE TITLE
TSDAE for Indonesian Language 

### ISSUE THREAD

**Issue by 99sbr** [UKPLab/sentence-transformers#1558]
Hello ,

I want to understand what changes i would have to do in order to pretrain TSDAE on bahasa (Indonesian) language. Will the tokenisation and deletion of token work for other languages even if i use 'multilingual' backbone.

**Comment by ScottishFold007** [comment]
You can change the tokenizer(word splitter) to one that is suitable for your country's language, and then it may not be cut by words, but by characters

**Comment by sbrvrm99** [comment]
i am training the model for 20 epochs, 10 mil sentences, weight decay 0.01 with 3e-5 learning rate. Is this acceptable given i am training on bahasa with indobert as backbone. Any improvement you can suggest here? 

Also how to i know when my model converges? can i print loss?

**Comment by nreimers** [comment]
Sounds good. When you change the fit function, you can also print the loss

**Comment by 99sbr** [comment]
I have not changed the fit function. These parameters are part of fit function. But in order to print loss what changes do i need to do?
